unreleased.md

```diff
@@ -3,3 +3,7 @@
 **Added:**
 - react-native-bpk-component-pagination-dots:
   - Introducing the React Native pagination dots component.
+
+- bpk-component-modal:
+  - Ability to remove padding from inner content.
+  - Ability to add a custom class name to inner content.
```

Comment:
Can we expand these two to mention how to do this?

Reply:
I went for:
```
- Ability to remove padding from inner content e.g. `padded={false}`.
- Ability to add a custom class name to inner content e.g. `contentClassName="my-custom-class"`.
```